docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Set up transactional xCluster replication
+headerTitle: Set up transactional xCluster replication
+linkTitle: Set up replication
+description: Setting up transactional (active-standby) replication between universes
+headContent: Set up unidirectional transactional replication
+menu:
+  preview:
+    parent: async-replication-transactional
+    identifier: async-transactional-setup-2-db
+    weight: 10
+badges: ysql
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../async-transactional-setup/" class="nav-link">
+      <i class="icon-shell"></i>
+      Manual Setup
+    </a>
+  </li>
+  <li >
+    <a href="../async-transactional-setup-dblevel/" class="nav-link active">
+      <i class="icon-shell"></i>
+      Database Level Setup
+    </a>
+  </li>
+</ul>
+
+Database-level setup of transactional xCluster replication simplifies the operational complexity of managing replication and making DDL changes.
+
+With database-level setup, xCluster replication operates at the YSQL database granularity. This means you only run xCluster management operations when adding and removing databases from replication, and not when tables in the databases are created or dropped.
+
+In particular, with database-level setup, [DDL changes](#making-ddl-changes) don't require the use of yb-admin. This means DDL changes can be made by any database administrator or user with database permissions, and don't require SSH access or intervention by an IT administrator.
+
+The following assumes you have set up Primary and Standby universes. Refer to [Set up universes](../async-deployment/#set-up-universes).
+
+## Set up replication
+
+To set up database-level unidirectional transactional replication, do the following:
+
```

Comment:
We still need to enable PITR the standby database. Same instructions from Manual mode can be copied here